src/test/java/uk/gov/companieshouse/pscstatement/delta/serialization/ChsDeltaSerializerTest.java

```diff
@@ -0,0 +1,68 @@
+package uk.gov.companieshouse.pscstatement.delta.serialization;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.companieshouse.delta.ChsDelta;
+import uk.gov.companieshouse.logging.Logger;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+@ExtendWith(MockitoExtension.class)
+public class ChsDeltaSerializerTest {
+
+    private String payload = "{\"data\": \"{\\n  \\\"company_number\\\": \\\"09950914\\\"," +
```

Comment:
Could we call the TestHelper to create this payload so we test against a delta? Maybe as well as a string as opposed to instead of